data/seriesRepository.go

```diff
@@ -740,7 +741,9 @@ func (r *SeriesRepository) GetTransformation(
 		return
 	}
 	var (
-		observations []models.DataPortalObservation
+		observation_dates  []string
```

Comment:
Let's stick to the pattern of camelCase or super short when the variable is declared and used within a small span of code (https://github.com/golang/go/wiki/CodeReviewComments#variable-names).